webapp-django/crashstats/crashstats/utils.py

```diff
@@ -389,16 +389,26 @@ def get_versions_for_product(product='Firefox', use_cache=True):
         return []
 
     # Get versions from facet, drop junk, and sort the final list
+    betas = set()
     versions = []
     for item in ret['facets']['version']:
         version = item['term']
         try:
             # This generates the sort key but also parses the version to
             # make sure it's a valid looking version
             versions.append((generate_version_key(version), version))
+
+            # Add X.Yb to betas set
+            if 'b' in version:
+                beta_version = version[:version.find('b') + 1]
+                betas.add(beta_version)
```

Comment:
This is nice to do here because this work will also get cached. So, in general, this is a good move.